### PR TITLE
Update Footer to be more responsive

### PR DESCRIPTION
**Pull request by 1CIM**
### Suggested changes
- Refactor component to use makeStyles 
- Make component more responsive
- Change names on test files

### Media
https://recordit.co/0x969bbNsS

### CODE REVIEW

File: src/components/ApplicationFooter.jsx [hunk 1]
```diff
@@ -3,17 +3,127 @@ import {
...
+  footerContainer: {
+    [theme.breakpoints.up("xs")]: {
+      display: "flex",
+      flexDirection: "column",
+      width: "90%",
+      maxWidth: "800px",
+      marginLeft: "auto",
+      marginRight: "auto",
+      position: "relative",
+      bottom: "0",
```
Comment (SevaDer14) [inline src/components/ApplicationFooter.jsx:23]: why don't have it on larger screens?

File: src/components/ApplicationFooter.jsx [hunk 2]
```diff
@@ -3,17 +3,127 @@ import {
...
+      marginBottom: "0px",
+      paddingTop: "20px",
+    },
+  },
+  hiddenScrollText: {
+    [theme.breakpoints.up("xs")]: {
+      top: "0",
+      left: "0",
+      bottom: "-40px",
+      right: "-20px",
```
Comment (SevaDer14) [inline src/components/ApplicationFooter.jsx:88]: that is interesting.. why negative?

**Review by SevaDer14 — APPROVED** [review]
Looks great! just couple of questions

File: src/components/ApplicationFooter.jsx (hunk 1 above)
Comment (1CIM) [reply src/components/ApplicationFooter.jsx:23]: when its on xs and nothing else it gets to everything thats above xs as well

File: src/components/ApplicationFooter.jsx (hunk 2 above)
Comment (1CIM) [reply src/components/ApplicationFooter.jsx:88]: Its to hide the scroll bars that appears because of the overflow and text wrap

**Pull request merged**